setup.py

```diff
@@ -99,7 +99,7 @@
     keywords="",
     install_requires=[
         "PyJWT==2.0.*",
```

Comment:
You should also consider relax for this.

Reply:
We should also freeze the version for this in `development` array above cause during dev, we will all then use the same version - not causing issues.